### DISCUSSION TITLE
Starting emails trough SL may become paid feature. This is the ways of solution

### DISCUSSION THREAD

**Discussion by CoderBoy675** [simple-login/app#1827] [Feature Requests]
Hello!

[This is](https://github.com/simple-login/app/blob/e9e863807cf1ba69db5dd16ed6ceda18957091fe/app/errors.py#L74) what I am talking about

And [this is](https://github.com/simple-login/app/discussions/1821#discussioncomment-6598658) the answer of SimleLogin founder

In this case I suggest the following solution to the problem that will be interesting for both sides:
- Make it possible to start emails only for accounts older than 3 months (this will prevent attackers from constantly registering new accounts)
- Allow Proton users to start emails in any case (provided Simplelogin account is linked to Proton account).
- Introduce a limit of 150 outgoing emails per day, similar to what Proton did.

But please do NOT make the start emails feature exclusively paid.

P.S: I already left a comment [HERE](https://github.com/simple-login/app/discussions/1821#discussioncomment-6599669) but I think it will more convenient if it will be a separate discussion

P.P.S: Please, be polite. Without any insults :)

**Comment by trikotret** [comment]
I like this solution very much. No reason to not implement it since free Proton Mail has 150 outgoing daily limit.

**Reply by HunterDecay** [reply]
I think that too. Unlimited sending a day cause abuses. I think FREE 150 outgoing emails per day will be enough for legitimate users

**Comment by Cutepanda56** [comment]
I really disappointed about this news… But I think this solutions will really help and be acceptable for both sides

**Comment by HunterDecay** [comment]
Dear developers, personally `@nguyenkims`. Please consider implementing **all** of this prevention measures. I hope this will solve our and your problems both. We a not your enemies, we are trying to make service better!

**Comment by VirtualPassive** [comment]
So… Previous discussion wiped again. Isn’t this a censorship?

Dear devs please provide the following solutions. DO NOT make it only paid.

**Comment by nguyenkims** [comment]
This is my answer in other post as it has been removed by our social team, probably because as the original author uses disrespectful languages.

```
This is a misunderstanding.

Recently, we have noticed an increase in the number of abusers who use Free SimpleLogin accounts to blackmail people, send spam, and perform similar actions prohibited by the the Terms of Service of both SimpleLogin and Proton. These actions have a negative effect on all SimpleLogin users as other email services might start to block emails sent from SimpleLogin.

As a countermeasure, we plan to only allow users with a Paid subscription to initiate a new email from an alias. Please note the following:

- Free users will still be able to reply with a reverse alias to an email previously sent to their SimpleLogin alias.

- This limit won't apply to existing Free users, only to new accounts created once the limit is implemented.

Also, please note that this forum is to discuss about SimpleLogin in a civil manner and we won't tolerate disrespectful messages, such as the ones you had sent in previous threads: any person who fails to do so will be banned. 
```


> Make it possible to start emails only for accounts older than 3 months (this will prevent attackers from constantly registering new accounts)

It can be indeed a good idea.

> Allow Proton users to start emails in any case (provided Simplelogin account is linked to Proton account).

Abuse is also a challenge on proton side so this wouldn't solve the issue.

> Introduce a limit of 150 outgoing emails per day, similar to what Proton did.

From what we see, abusers send targeted blackmails to people they know, so it isn't a spam campaign and they don't need to send out a lot of emails, they just need the From address to be different each time so the victim has no way to block them. Limiting the number of outgoing emails unfortunately won't work for that case.

That being said, making this feature paid isn't the silver bullet solution either but it greatly discourages abusers.

This limit isn't enforced at the moment. We also mentioned it in the pricing page to make sure this isn't a surprise for new users.

![CleanShot 2023-08-01 at 19 15 20@2x](https://github.com/simple-login/app/assets/3167780/861c69a9-e221-40d4-b48d-bb63ea7decc1)

As the number of SL users grows very quickly, the number of abusers increase as well and without a strict measure, it might end up affecting us all. 
Most users use SL for receiving emails so we don't expect any financial increase with this limit: its only goal is to fight against abusers.  If the abuser issue is stabilized, we can remove that limit.

**Reply by Bored-pandao-alias3** [reply]
I am sorry for being disrespectful but I’m really pissed off.

Yes the account from what I am writing now is my new account because of you ban.

So:
- Will it affect old users? Or only new?
- What about to block sending for accounts that had been registered too recently and allow for accounts that had a year or more?

**Reply by nguyenkims** [reply]
> Will it affect old users? Or only new?

Only new accounts, existing accounts are grandfathered in.

> What about to block sending for accounts that had been registered too recently and allow for accounts that had a year or more?

It's indeed a good idea.

**Comment by WeboR6** [comment]
I think that this is really bad idea. Dear developers, please, try to block sending feature for accounts that have been registered less than 3 months ago. I hope this will solve the issue.

**Comment by Raekye** [comment]
Let's keep in mind that - assuming/hoping that Simple Login isn't selling our data, they're providing a... _truly free_ service. They don't get _anything_ out of non-paying users (please correct me if I'm wrong).

Of course, most people are (presumably) honest users, and if there's an alternative, e.g. as proposed here, where SL can continue providing useful features to free users, SL would be a great company if they did so. But if SL chooses the "easy" path of moving features to the paid service only, I don't think there's any fair argument to be made against them. It's their service. (In fact, since they've made it open source and even [documented quite well the process of self-hosting SL](https://github.com/simple-login/app/blob/master/README.md), nothing is stopping anyone from running SL themselves - except, well, the fact that running servers costs time and money 😅)

**Reply by Lover563** [reply]
> Of course, most people are (presumably) honest users, and if there's an alternative, e.g. as proposed here, where SL can continue providing useful features to free users, SL would be a great company if they did so.

That’s the only what really important. The most terrible is that, they deleting all criticism discussions and comments. I saw a screenshot on Reddit with proofs and I am here to “tell my 2 pennies”.

My opinion is - what is free now must be free forever. They can add more useful (new) features to paid users only but they mustn’t steal what is already available. Period.

No hateful comments. Only IMHO

**Reply by Raekye** [reply]
> My opinion is - what is free now must be free forever.

I'd like to live in a world where things don't cost money, but I don't think that's our world... SL did something nice by offering free services (again, assuming/hoping they didn't sell our data), but, uh, I fail to see why they (or anyone) must continue to do stuff for free for other people, just because they initially did it for free?

> They can add more useful (new) features to paid users only but they mustn’t steal what is already available. Period.

Can you show me the part where they took something away that belonged to someone else? Like, you know, **someone bought something and SL took it away from them**, or someone made something and SL took it away from them?

> The most terrible is that, they deleting all criticism discussions and comments.

I haven't been around discussions around to really know what's going on here. If they are silently censoring discussion, that's terrible.

From what I can see (I just dropped in to GitHub recently), this thread has been up for 2 weeks, and the developer claimed that the social team removed the original thread due to disrespectful language (I'm not saying this claim can be trusted). Personally, SL seems pretty transparent - e.g. with their pricing page.

Also my personal opinion, but while the language in this thread isn't outright disrespectful, some users have a... _tone_ that I wouldn't ever want to deal with; even if they were paying customers, if I were running a business, I'd rather just not have these customers 😅

**Reply by Timm346** [reply]
I think that it is not whole picture. And they are censoring thread indeed.

I made an investigation in another comment below.

**Comment by Timm346** [comment]
I read the whole thread and many anger posts on Reddit and twitter. So I really think that developers is wrong right there. They should block sending to specific address (of person who has been blackmailed) or provide a limit for sending.

Why paid sending **WON’T** solve the problem **AT ALL**:

**_If you create new simplelogin account you will have a 7-day trial period that provides you a temporary access to ALL paid features._** 

So why an abuser can’t create more and more accounts? All of new accounts will have a trial period. And that’s all what abuser need.

Also abuser can remove SL account and create it again for the same Email address to renew trial period.

As @nguyenkims said [here](https://github.com/simple-login/app/discussions/1827#discussioncomment-6608305):
>they just need the From address to be different each time so the victim has no way to block them. Limiting the number of outgoing emails unfortunately won't work for that case

So the only solution is to limit sending feature to accounts that had been registered less than 3 months ago. After 3 months SL should automatically enable sending feature FOR FREE. 

I think developers hiding something in this situation. Because after small investigation it looks like everyone can bypass feature “sending” restriction.

**PLEASE DO NOT USE MY INSTRUCTION FOR MALICIOUS PERPOSES. It had been provided only as proof, not as a scammer manual**

For example:
1. Go to [this link](https://emkei.cz)
2. Then in “From E-mail” enter the email for what you want to send potentially abusive email.
3. In “To” field enter your email alias
4. Subject, Name and text fields can be random
5. Send email

After this, mail will be marked that it failed anti phishing checks (and will probably got YOUR to SPAM folder) but you will be able to “reply” to potential victim.

So I think developers don’t really have abuse problems and they probably hiding some motivation.

I hope this will help many people who are discussing here.

PS: This investigation took me 1,5 hours to be done.

**Reply by nguyenkims** [reply]
> So why an abuser can’t create more and more accounts? All of new accounts will have a trial period. And that’s all what abuser need.

The free trial gives access to some premium features but not all. For ex, you can't create more than 10 aliases. When the limit is live, we'll probably do the same to avoid this workaround.

> PLEASE DO NOT USE MY INSTRUCTION FOR MALICIOUS PERPOSES. It had been provided only as proof, not as a scammer manual

The workaround you mentioned might work today but can be blocked anytime if the service you use is blocked. It will also probably fail the anti-spam layer in SimpleLogin anyway.

> So I think developers don’t really have abuse problems and they probably hiding some motivation.

I don't think it's worth discussing if you decide to jump to this conclusion based on the "investigation" you made. Accusations are rarely helpful in the discussion.

Again, as I mentioned in https://github.com/simple-login/app/discussions/1827#discussioncomment-6608305, you won't be affected if you already have a SL account.

> - Free users will still be able to reply with a reverse alias to an email previously sent to their SimpleLogin alias.

> - This limit won't apply to existing Free users, only to new accounts created once the limit is implemented.

**Comment by nguyenkims** [comment]
Hi as the discussion here seems to be less and less constructive, I'll lock the discussion.  

Just a reminder that you won't be affected if you already have a SL account.

If you're unhappy with this decision, you can self host SimpleLogin or use another service.
